extra/exports

```diff
@@ -420,6 +420,8 @@ tnt_datetime_strftime
 tnt_datetime_to_string
```

Comment:
The patch looks good now! However a couple of moments to polish:
```
tarantool> i = interval.new({month = math.huge})
---
...

tarantool> i
---
- -2147483648 months
...
```
```
tarantool> i = interval.new({month = box.NULL})
---
- error: 'builtin/datetime.lua:291: cannot convert ''void *'' to ''int'''
...
```
```
tarantool> i = interval.new({month = 999999999999999})
---
...

tarantool> i
---
- -2147483648 months
...
```

Reply:
After (approximated) range check introduced for interval values we have this modified behaviour for cases above:

```
tarantool> I = date.interval.new{month = math.huge}
---
- error: '[string "I = date.interval.new{month = math.huge}"]:1: value inf of month
    is out of allowed range [-141110652, 141110652]'
...

tarantool> I = date.interval.new{month = box.NULL}
---
...

tarantool> I
---
- 0 seconds
...

tarantool> I = date.interval.new{month = 999999999999999}
---
- error: '[string "I = date.interval.new{month = 999999999999999}"]:1: value 1e+15
    of month is out of allowed range [-141110652, 141110652]'
...

```